atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/OracleTableNameUnmapper.java

```diff
@@ -51,7 +51,7 @@ public String getShortTableNameFromMappingTable(
             return unmappingCache.get(fullTableName, () -> {
                 SqlConnection conn = null;
                 try {
-                    conn = connectionSupplier.getNewUnsharedConnection();
+                    conn = connectionSupplier.get();
```

Comment:
For some reason I thought that we wrap the connection into some sort of reference-counting wrapper, apparently we don't. I will remove the close() calls. I'm wondering how the tests are passing on circleci, though.